src/connectdlg.cpp

```diff
@@ -237,7 +237,7 @@ void CConnectDlg::RequestServerList()
     }
     cbxDirectoryServer->blockSignals ( false );
 
-    // Get the IP address of the directory server (using the ParseNetworAddress
+    // Get the IP address of the directory (using the ParseNetworAddress
```

Comment:
```suggestion
    // Get the IP address of the directory (using the ParseNetworkAddress
```

I have not checked if the function name does omit the k.

Reply:
Good spot, if a bit late ;)